chans/kdspub/kdspub.go

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2021 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package kdspub
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/Comcast/plax/dsl"
+	"github.com/aws/aws-sdk-go-v2/aws"
```

Comment:
Please add a newline between the `plax` import and the AWS SDK imports. Having multiple blocks like that improves readability (and that's the only benefit AFAIK).

Reply:
Done